tests/test_config.py

```diff
@@ -188,6 +244,16 @@ def testMultipleConfigFiles(self):
 password=password
 owner=root
 env=staging
+rhsm_username=rhsm_admin1
+rhsm_password=rhsm_password1
+rhsm_hostname=host1
+rhsm_ssl_port=12341
+rhsm_prefix=prefix1
+rhsm_proxy_hostname=proxyhost1
+rhsm_proxy_port=43211
+rhsm_proxy_user=proxyuser1
+rhsm_proxy_password=proxypass1
+rhsm_insecure=1
```

Comment:
Yes. As of the most recent commit, the values in rhsm.conf are used as the basis, and those provided in the host config override the rhsm.conf "defaults."